CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## v1.4.1 [unreleased]
```

Comment:
Since we're just doing a quick patch release for this, can you go ahead and set the date to 2017-11-13?